src/main/scripts/package.json

```diff
@@ -4,84 +4,110 @@
   "description": "Lumeer Web UI",
   "main": "index.js",
   "scripts": {
-    "build": "webpack -p",
+    "ng": "ng",
+    "i18n": "ng xi18n --output-path src/i18n",
     "start": "cross-env LUMEER_ENV=development webpack-dev-server",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack",
```

Comment:
Shouldn't we use Angular CLI commands instead? Or why do we use `webpack` and `webpack-dev-server` here?

Reply:
We can't use angular cli for build/start and serve commands anymore, since we changed our webpack config (which has to be done, because of our settings of proxy and such). However most (probably all we need) of commands are available in script part of `package.json`

Here is described how it has been done and what it is good for https://github.com/angular/angular-cli/issues/6302